app/src/org/odk/collect/android/widgets/BarcodeWidget.java

```diff
@@ -44,15 +45,16 @@
 
     private TextView mStringAnswer;
 
-    public BarcodeWidget(Context context, FormEntryPrompt prompt, Intent i, IntentCallout ic) {
-        super(context, prompt, i, ic, FormEntryActivity.BARCODE_CAPTURE);
+    public BarcodeWidget(Context context, FormEntryPrompt prompt, Intent i, IntentCallout ic,
```

Comment:
can you add a TODO to just mention that we aren't using this yet so whoever implements the Pending Callouts for images and stuff knows to include this widget  as well?